solr/solr-ref-guide/modules/configuration-guide/pages/implicit-requesthandlers.adoc

```diff
@@ -96,15 +85,6 @@ This handler must have a collection name in the path to the endpoint.
 |`solr/<collection>/admin/ping` |{solr-javadocs}/core/org/apache/solr/handler/PingRequestHandler.html[PingRequestHandler] |`_ADMIN_PING`
 |===
 
-Plugins:: Return info about all registered plugins.
```

Comment:
removed; we'll ensure this removal is stated elsewhere for Solr 10 elsehwere.  Found via broken link detector